packages/aws/src/RPA/Cloud/AWS/__init__.py

```diff
@@ -100,14 +106,15 @@ def _init_client(
         aws_key: str = None,
         region: str = None,
         use_robocloud_vault: bool = False,
+        session_token: str = None,
```

Comment:
```suggestion
        session_token: Optional[str] = None,
```
here and everywhere else.

Reply:
did a regex replace throughout to resolve